UPGRADE.md

```diff
@@ -1,5 +1,40 @@
 # Upgrade
 
+## dev-release/2.0
+
+### DeleteToolbarAction with conflict
+
+The `DeleteToolbarAction` asks for confirmation if e.g. a page that is being tried to deleted is linked on other pages.
```

Comment:
not sure if `being tried to deleted` is correct. but dont have a better suggestion and i guess people will understand it anyway 🙂